python/py-logbook/Portfile

```diff
@@ -8,10 +8,9 @@ python.rootname     Logbook
 version             1.5.3
 revision            0
 categories-append   devel
-platforms           darwin
 license             BSD
 
-python.versions     27 35 36 37 38 39
+python.versions     27 37 38 39
```

Comment:
Is there any reason to keep py27-logbook subport?

Reply:
yes, it still has a dependent, for which I didn't manage (yet) to update the Python version: 
```
% port echo depends:py27-logbook
pyfa
```